jib-core/src/main/java/com/google/cloud/tools/jib/api/JavaContainerBuilder.java

```diff
@@ -204,7 +204,7 @@ public static JavaContainerBuilder from(TarImage tarImage) {
   private RelativeUnixPath dependenciesDestination = RelativeUnixPath.get("libs");
   private RelativeUnixPath othersDestination = RelativeUnixPath.get("classpath");
   @Nullable private String mainClass;
-  private ModificationTimeProvider modificationTimeProvider =
+  private static ModificationTimeProvider modificationTimeProvider =
```

Comment:
This should remain an instance variable (non-`static`), scoped and limited only to each individual instance. `static` means a global variable shared by every instance of `JavaContainerBuilder`.